src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleCss = sourceString.split(';')
+    .filter((style) => style.includes(':'))
+    .map((style) => style.split(':'));
+
+  const resultStyle = (object, style) => {
```

Comment:
Назва функції має починатися з дієслова
```suggestion
  const addProperty = (styles, property) => {
```

Reply:
Можеш використати деструктуризацію
```suggestion
  const resultStyle = (object, [key, value]) => {
```